packages/sources/tp-icap/src/config/includes.json

```diff
@@ -0,0 +1,2895 @@
+[
```

Comment:
An alternative to this, if we are confident, is to simply perform inverses for any pair with `USD` as the base. Open to thoughts.

Reply:
Going ahead with the change here since it's much more tenable that having hundreds of includes overrides